mne/channels/tests/test_channels.py

```diff
@@ -344,6 +344,18 @@ def test_find_ch_adjacency():
     assert ch_names[0] == 'MEG 001'
 
 
+@testing.requires_testing_data
+def test_neuromag122_adjacency():
+    """Test computing the adjacency matrix of Neuromag122-Data"""
```

Comment:
pydocstyle is unhappy
```suggestion
    """Test computing the adjacency matrix of Neuromag122-Data."""
```